app/src/lib/text-token-parser.ts

```diff
@@ -310,15 +332,25 @@ export class Tokenizer {
           )
           break
 
-        case 'h':
+        default:
```

Comment:
Wondering if this would be easier to follow with less nested blocks and some docs?

```
        default:
          // Hyperlinks and external issue could start with any letter so need to check both.
          const oldi = i
          i = this.inspectAndMove(element, i, () =>
            this.scanForExternalIssue(text, i, repository)
          )

          // if we found an external issue, we don't want to continue scanning
          if (oldi !== i) {
            break
          }

          if (element === 'h') {
            i = this.inspectAndMove(element, i, () =>
              this.scanForHyperlink(text, i, repository)
            )
            break
          }

          // No special token found, just append the character
          this.append(element)
          i++

          break
      }
```